app/views/includes/head.jade

```diff
@@ -11,8 +11,9 @@ head
   meta(name="apple-mobile-web-app-capable", content="yes")
   meta(name="viewport", content="initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=0")
 
-  link(href='/img/icons/favicon.ico', rel='shortcut icon', type='image/x-icon')
-  link(rel="stylesheet", href="https://fonts.googleapis.com/icon?family=Material+Icons")
+  link(href='https://res.cloudinary.com/dqsmurjpg/image/upload/v1530719572/Pasted_image_at_2018_07_03_01_02_PM.jpg', rel='shortcut icon', type='image/x-icon')
```

Comment:
Rename the file to something more appropriate

Reply:
Done